doc/source/whatsnew/v0.21.0.txt

```diff
@@ -112,6 +112,7 @@ Plotting
 
 Groupby/Resample/Rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
+- Bug in ``resample().size()``. Inconsistent return type on resample of empty DataFrame (:issue:`14962`)
```

Comment:
Bug in ``DataFrame.resample().size()`` where an empty DataFrame did not return a Series.